pkgs/by-name/co/color-scheme-generator/package.nix

```
@@ -0,0 +1,44 @@
+{
+  rustPlatform,
+  fetchFromGitHub,
+  lib,
+  perl,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "color-scheme-generator";
+  version = "0.9.2.1";
+
+  src = fetchFromGitHub {
+    owner = "nikolaizombie1";
+    repo = "color_scheme_generator";
+    rev = "v${version}";
+    hash = "sha256-tSoM6MP58vOoKRMQjNq8nKMiVTfS9N/mckchAPcUM0g=";
+  };
+
+  cargoHash = "sha256-A8ld2c8ZcqaZN29Fz085FQy8UpugIDMOKT01r+SfCcc=";
+
+  nativeBuildInputs = [ perl ];
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
Nitpick, but we try to avoid larger `with ...;` scopes like this